src/main/java/httpserver/HttpPresenter.java

```diff
@@ -0,0 +1,12 @@
+package httpserver;
+
+public class HttpPresenter implements Presentable{
+
+    public String present() {
+        String responseLine = "HTTP/1.1 405 Method Not Allowed" + System.lineSeparator();
+        String header = "Allow: HEAD, OPTIONS" + System.lineSeparator();
+        String blankLine = System.lineSeparator();
```

Comment:
Cool! I didn't know there was a `System.lineSeparator()` for getting the line separator string and its system dependent 😎 What does it return for you?

Reply:
I didn't know about it either until I started reading about CRLF.  Using System.lineSeparator() is platform independent.  See article:  https://www.baeldung.com/java-string-newline

Reply:
Thanks for sharing! I learned something new 😄 That CRLF was tricky when I was working on my server project